__test__/backend.test.js

```diff
@@ -204,34 +206,39 @@ it('should convert an invitation and user into a valid organization instance', a
 
 
 it('should create a test campaign', async () => {
-
-  const [user, invite] = await Promise.all([createUser(), createInvite()])
-  const organization = await createOrganization(user, "test org", user.id, invite.data.createInvite.id)
-
   const campaignTitle = "test campaign"
-  const campaign = await createCampaign(user, campaignTitle, "test description", organization.data.createOrganization.id)
-  expect (campaign.data.createCampaign.title).toBe(campaignTitle)
+  testCampaign = await createCampaign(testAdminUser, campaignTitle, "test description", testOrganization.data.createOrganization.id)
 
+  expect (testCampaign.data.createCampaign.title).toBe(campaignTitle)
 })
 
-
 it('should create campaign contacts', async () => {
-  const [user, invite] = await Promise.all([createUser(), createInvite()])
-  const organization = await createOrganization(user, "test org", user.id, invite.data.createInvite.id)
-
-  const campaign = await createCampaign(user, "test campaign", "test description", organization.data.createOrganization.id)
-
-  const contact = await createContact(campaign.data.createCampaign.id)
-
-  expect (contact.campaign_id).toBe(parseInt(campaign.data.createCampaign.id))
+  const contact = await createContact(testCampaign.data.createCampaign.id)
 
+  expect (contact.campaign_id).toBe(parseInt(testCampaign.data.createCampaign.id))
 })
 
-// it('should add texters to a organization', async () => {
-//
-//
-//
-// })
+it('should add texters to a organization', async () => {
+  testTexterUser = await createUser({
+    auth0_id: 'test456',
+    first_name: 'TestTexterFirst',
+    last_name: 'TestTexterLast',
+    cell: '555-555-6666',
```

Comment:
not sure we use this anywhere, but are `-`s allowed? (or is that part of the test)?